site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2433,6 +2433,7 @@ public function getGradedComponentsCountByGradingSections($g_id, $sections, $sec
             $where = "WHERE active_version > 0 AND ({$section_key} IN " . $this->createParameterList(count($sections)) . ") IS NOT FALSE";
             $params = array_merge($params, $sections);
         }
+
```

Comment:
This whitespace change should be reverted.

Reply:
fixed in 22d3334b28ff3cde8f82c1bbb520701d377371ef